### PR TITLE
roachtest: update pgjdbc blocklists

### LINKED ISSUE #53467: roachtest: pgjdbc failed
[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2223072&tab=buildLog) on [master@ef55609797c92e46eb9efd08facc9db49558291d](https://github.com/cockroachdb/cockroach/commits/ef55609797c92e46eb9efd08facc9db49558291d):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/pgjdbc/run_1
	orm_helpers.go:218,orm_helpers.go:144,java_helpers.go:216,pgjdbc.go:175,pgjdbc.go:188,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2782-gef55609797
		Tests run against pgjdbc REL42.2.9
		5882 Total Tests Run
		4957 tests passed
		925 tests failed
		50 tests skipped
		4 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testBadColumnIndexes - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testZeroRowResult - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testUpdateablePreparedStatement - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testDeleteRows - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testMultiColumnUpdateWithoutAllColumns - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.UpdateableResultTest.testInsertRowIllegalMethods - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>

Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2223072&tab=artifacts#/pgjdbc)
Related:
- #53391 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008241848_v20.1.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008241848_v20.1.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #53255 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

### LINKED ISSUE #53738: roachtest: pgjdbc failed
[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2237018&tab=buildLog) on [release-20.1@ff3749579c10bb948e89598a5b58bfdcfeb432be](https://github.com/cockroachdb/cockroach/commits/ff3749579c10bb948e89598a5b58bfdcfeb432be):

```
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedNoArgBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testNoServerPrepareOneRow[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchReturningMixedNulls[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testEmptyClearBatch[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testEmptyClearBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes1[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenSpaceAfterValues[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes3[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testTransactionalBehaviour[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes0[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes1[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes2[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testEmptyClearBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedNoParameters[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithTwoMultiInsertStatements[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes2[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSmallBatchUpdateFailureSimple[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testServerPrepareMultipleRows[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchEscapeProcessing[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithTwoMultiInsertStatements[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSmallBatchUpdateFailureSimple[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchEscapeProcessing[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_1) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>

Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2237018&tab=artifacts#/pgjdbc)
Related:
- #53467 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53255 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

### LINKED ISSUE #54106: roachtest: pgjdbc failed
[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254937&tab=buildLog) on [release-20.2@d9ff57ed3d5891ab93b7756be657ba868ecc87a6](https://github.com/cockroachdb/cockroach/commits/d9ff57ed3d5891ab93b7756be657ba868ecc87a6):

```
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedNoParameters[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenMixedCaseValues[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testTransactionalBehaviour[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithTwoMultiInsertStatements[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes2[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testServerPrepareMultipleRows[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes5[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testServerPrepareMultipleRows[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes1[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testNoServerPrepareOneRow[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithMultiInsert[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testWarningsAreCleared[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testTransactionalBehaviour[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- PASS: org.postgresql.test.jdbc4.jdbc41.SchemaTest.testUsingSchema - https://github.com/cockroachdb/cockroach/issues/26443 (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes6[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes3[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchEscapeProcessing[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenSpaceAfterValues[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testNoServerPrepareOneRow[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testEmptyClearBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithMultiInsert[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testTransactionalBehaviour[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchReturningMixedNulls[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes0[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes3[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenSpaceAfterValues[2: autoCommit=NO, binary=REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithTwoMultiInsertStatements[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSmallBatchUpdateFailureSimple[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithTwoMultiInsertStatements[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>

Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254937&tab=artifacts#/pgjdbc)
Related:
- #53738 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

- #53467 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53255 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

### PR DESCRIPTION
**Pull request by solongordon**
I made the following updates to the pgjdbc blocklists:
- Removed tests which are now passing due to user-defined schema
  support.
- Removed testUpdateSelectOnly since we now support this syntax as a
  no-op.
- Updated some failure reasons to "unknown" for tests which are still
  failing even though the referenced issue was closed.
- Added many BatchExecuteTest.* tests to the ignore list. These tests
  are flaky due to a combination of #54477 and the fact that the tests
  do not run in a deterministic order.

Fixes #53467
Fixes #53738
Fixes #54106

Release note: None

### CODE REVIEW

**Review by rafiss — CHANGES_REQUESTED** [review]
lgtm! just a small note about removing the 19.1 stuff

also, just FYI, i believe this will need to backported to each release branch once it is merged. #51897 changed which roachtest binary is used in nightlies

File: pkg/cmd/roachtest/pgjdbc_blocklist.go [hunk 1]
```diff
@@ -6333,16 +6327,129 @@ var pgjdbcIgnoreList20_2 = pgjdbcIgnoreList20_1
...
 
-var pgjdbcIgnoreList19_2 = blocklist{
-	"org.postgresql.replication.ReplicationTestSuite.org.postgresql.replication.ReplicationTestSuite": "expected fail - no replication",
-	"org.postgresql.test.core.LogServerMessagePropertyTest.testWithDefaults":                          "expected fail - checks error message",
-	"org.postgresql.test.core.LogServerMessagePropertyTest.testWithExplicitlyEnabled":                 "expected fail - checks error message",
-	"org.postgresql.test.core.LogServerMessagePropertyTest.testWithLogServerErrorDetailDisabled":      "expected fail - checks error message",
-}
+var pgjdbcIgnoreList19_2 = pgjdbcIgnoreList19_1
 
 var pgjdbcIgnoreList19_1 = blocklist{
```
[the remark this answers is not in the document]
Comment (solongordon) [reply pkg/cmd/roachtest/pgjdbc_blocklist.go]: Done. I got rid of all the <19.2 lists.

**Comment by solongordon** [comment]
Thanks for the heads up about backports!

**Review by rafiss — APPROVED** [review]

**Pull request merged**